tap-gui/plugins/runtime-resource-visibility.hbs.md

```diff
@@ -138,12 +158,21 @@ Users can see the pod table in each resource details page.
 
 ### <a id="pod-details-metrics"></a> Overview of pod metrics
 
-The overview card displays the user-configured resource limits on the pod, defined in accordance with
-the [Kubernetes documentation](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/).
-These limits do not represent actual real-time resource use. To monitor actual real-time resource use,
-see [Application Live View for Spring Boot Applications in Tanzu Application Platform GUI](app-live-view-springboot.md).
+If you have a metric server running on your cluster, you will find that the
+overview card displays realtime metrics of your pod presented as a percentage
+of the max capability of this one.
+
+In the scenario where you have no metric server, the overview card displays the
+user-configured resource limits on the pod, defined in accordance with the
+[Kubernetes documentation](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/).
+
+For applications built using Spring Boot, you can also monitor the actual
+real-time resource usage using: 
+[Application Live View for Spring Boot Applications in Tanzu Application Platform GUI](app-live-view-springboot.md).
 
-Each container displays its resource limits, if defined.
+Each container displays its resource limits, if defined. Also a warning icon will
+be displayed if those limits are going to conflic someway with the `LimitRange`
+present on the namespace if any. (The warning icon will provide information about the conflic)
```

Comment:
Metrics and limits will also be displayed for each container on a pod details page. If a particular container's current limit conflicts with a namespace-level `LimitRange` (typically due to creating a container prior to applying a `LimitRange`), a small warning indicator will be displayed next to the container limit.

Reply:
Addressed, can you please re-review it? @menehune23

Reply:
Addressed, please consider resolved.